ScreenControl_Unity/Assets/ScreenControl/Service/WebSocketClientConnection.cs

```diff
@@ -53,16 +53,16 @@ private void InitialiseServer()
         {
             websocketInitalised = false;
 
+            receiverQueue = GetComponent<WebSocketReceiver>();
+            receiverQueue.SetWSClientConnection(Instance);
```

Comment:
This is not required if we are a singleton.

Reply:
We actually aren't using that reference once it's passed any more so I'm just gonna strip it